src/test/java/com/google/api/codegen/testdata/python_smoke_test_library.baseline

```diff
@@ -0,0 +1,47 @@
+============== file: test/google/cloud/gapic/example/library/v1/library_service_smoke_test.py ==============
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Smoke tests."""
```

Comment:
Our terminology for these kinds of tests in google-cloud-python is "system tests". This terminology is also shared with the DPE team for testing samples. Why choose this terminology instead?

Also, a nit: we typically omit module-level docstrings in test files.